tests/integration_test.py

```diff
@@ -588,242 +593,373 @@ def test_get_phenomena_by_name():
-        subscription = '3JSM4MN89SJFLE7VR6KPCM0BVPXTWT'
-        session = _get_sandbox_session()
-        query = Query().subsc.by_user(user).filter_by(subscription).around(5, '20200105', '20200115')
+        session = _get_dev_session()
+        query = Query().subsc.by_status(1)
 
         # when
-        entity = Context(session).get(query)
+        res = Context(session).get(query)
 
         # then
-        assert isinstance(entity, BinaryData)
-        assert subscription in entity.name
-        assert len(entity.content) > 0
+        self.assertIsInstance(res, Success, res.__str__())
+        self.assertTrue(len(res.result) > 0)
+        for subscription in res.result:
+            self.assertIsInstance(subscription, Subscription)
+            self.assertEqual(1, subscription.status)
+            print(subscription.code, 'from', subscription.owner, 'in', subscription.link)
+
+        print('\n')
+
+    def test_subscriptions_by_user_with_active_status(self):
+        print('Testing Subscriptions ========================')
+
+        # given
+        session = _get_dev_session()
+        query = Query().subsc.by_user(self._user).by_status(1)
+
+        # when
+        res = Context(session).get(query)
+
+        # then
+        self.assertIsInstance(res, Success, res.__str__())
+        for subscription in res.result:
+            self.assertIsInstance(subscription, Subscription)
+            self.assertEqual(1, subscription.status)
+            print(subscription.code, 'from', subscription.owner, 'in', subscription.link)
+
+        print('\n')
+
+    def test_subscriptions_by_user_get_dates(self):
+        print('Testing Subscriptions ========================')
 
-    @staticmethod
-    def test_last():
         # given
-        user = 'ensoag'
-        subscription = '3JSM4MN89SJFLE7VR6KPCM0BVPXTWT'
-        session = _get_sandbox_session()
-        query = Query().subsc.by_user(user).filter_by(subscription).last
+        session = _get_dev_session()
+        query = Query().subsc.by_user(self._user).by_key(self._subscription).get_dates()
 
         # when
-        entity = Context(session).get(query)
+        res = Context(session).get(query)
 
         # then
-        assert isinstance(entity, BinaryData)
-        assert subscription in entity.name
-        assert len(entity.content) > 0
+        self.assertIsInstance(res, Success, res.__str__())
+        print(res.__str__())
+        print('\n')
+
+    def test_subscriptions_by_user_with_dates(self):
+        print('Testing Subscriptions ========================')
 
-    @staticmethod
-    def test_last_n():
         # given
-        user = 'ensoag'
-        subscription = '3JSM4MN89SJFLE7VR6KPCM0BVPXTWT'
-        session = _get_sandbox_session()
-        query = Query().subsc.by_user(user).filter_by(subscription).last_n(5)
+        session = _get_dev_session()
+        query = Query().subsc.by_user(self._user).by_key(self._subscription).with_dates('20200519', '20200622')

```

Comment:
shouldn't it be  "...by_key(self._subscription).**dates.period**('20200519', '20200622')"  ????  idem to results.period

Reply:
lo mismo para la url entonces no?